### PR TITLE
Docs: updated developer guide

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
Improved installation instructions for windows.

### CODE REVIEW

File: contribute/developer-guide.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ brew install node@14
 npm install -g yarn
 ```
 
+### windows
```
Comment (oddlittlebird) [inline contribute/developer-guide.md:28]:
```suggestion
### Windows
```

**Review by oddlittlebird — APPROVED** [review]
One minor edit, then good to go!

File: contribute/developer-guide.md (hunk 1 above)
Comment (thisisobate) [reply contribute/developer-guide.md]: Done! 🤗

Comment (thisisobate) [reply contribute/developer-guide.md]: Thanks, @oddlittlebird for the corrections!

**Comment by thisisobate** [comment]
@oddlittlebird Seems like CI is broken...Let me see if I can notify the team about this

**Comment by hugohaggmark** [comment]
Friendly reminder to add labels, project and milestone to your PRs @thisisobate. Also, what is preventing this PR from being merged?

**Review by marcusolsson — APPROVED** [review]

**Comment by thisisobate** [comment]
None I can think of @hugohaggmark 

@oddlittlebird Can you take a look at this whenever you're free? Also, I added some comments to the docs so feel free to take a look as well.

File: contribute/developer-guide.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ brew install node@14
 npm install -g yarn
 ```
 
+### Windows
+
+If you are running on Windows 10, we recommend installing the Windows Subsystem for Linux (WSL). For installation instructions, refer to the [Microsoft WSL Installation Guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
```
Comment (achatterjee-grafana) [inline contribute/developer-guide.md:30]:
```suggestion
If you are running Grafana on Windows 10, we recommend installing the Windows Subsystem for Linux (WSL). For installation instructions, refer to the [Microsoft WSL Installation Guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10). 
```

**Review by achatterjee-grafana — APPROVED** [review]
Request for one change. Otherwise LGTM

**Comment by oddlittlebird** [comment]
I responded to the comments in the docs. We are waiting for a user to finish testing these instructions to see if we need to clarify anything before merging. There might be a virualization dependency, but Uche is investigating.

**Pull request merged**